ivy/functional/ivy/utility.py

```diff
@@ -47,11 +47,11 @@ def all(
     ----------
     x
         input array.
-    axis
+    axis.
         axis or axes along which to perform a logical AND reduction. By default, a
         logical AND reduction must be performed over the entire array. If a tuple of
         integers, logical AND reductions must be performed over multiple axes. A valid
-        ``axis`` must be an integer on the interval ``[-N, N)``, where ``N`` is the rank
+        ``axis`` must be an integer on the interval ``[-N, N]``, where ``N`` is the rank
```

Comment:
This correction isn't valid because by default the axes argument of numpy only allows out of order indices in the negative sense, so  `np.all(x,axes=-4)` is valid but `np.all(x,axes=4)`  is not valid.
assuming x has a shape of `(d1,d2,d3,d4)` 

Please revert this correction